.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "extern/ranges-v3"]
```

Comment:
Would leaving the submodules as an alternative cause in problems?

Reply:
Nope. It would work fine. However I am personally not a fan of submodules. They easily cause trouble and can become extremely cumbersome to maintain, that is why I deleted them.

Reply:
But if you would deliberately like not to have the submodules deleted you can cherry-pick this commit easily. I made sure all changes are atomic 😃